src/Mvc/Mvc.Abstractions/src/ModelBinding/ModelMetadata.cs

```diff
@@ -401,6 +490,11 @@ public virtual ModelMetadata ContainerMetadata
         /// </summary>
         public abstract Action<object, object> PropertySetter { get; }
 
+        /// <summary>
+        /// Gets a delegate that invokes the constructor.
+        /// </summary>
+        public virtual Func<object[], object>? ConstructorInvoker => null;
```

Comment:
Suggest `BoundConstructorInvoker` to be more clear and to align with src/Mvc/Mvc.Core/src/ModelBinding/Metadata/DefaultMetadataDetails.cs